docs/Reference/Hooks.md

```diff
@@ -267,6 +268,28 @@ service (if the `connectionTimeout` property is set on the Fastify instance).
 The `onTimeout` hook is executed when a request is timed out and the HTTP socket
 has been hanged up. Therefore, you will not be able to send data to the client.
 
+### onRequestAbort
+
+```js
+fastify.addHook('onRequestAbort', (request, reply, done) => {
+  // Some code
+  done()
+})
+```
+Or `async/await`:
+```js
+fastify.addHook('onRequestAbort', async (request, reply) => {
+  // Some code
+  await asyncMethod()
+})
+```
+`onRequestAbort` is useful if you need to monitor the if the client aborts the request
+(if the `request.raw.aborted` property is set to true).
+The `onRequestAbort` hook is executed when a client closes the connection before
+the entire request has been received. Therefore, you will not be able to send
+data to the client.
```

Comment:
```suggestion
The `onRequestAbort` hook is executed when a client closes the connection before
the entire request has been received. Therefore, you will not be able to send
data to the client.
```
I think the first sentence is redundant, as the second sentence is repeating it in a more formal tone.